CHANGES.md

```diff
@@ -8,6 +8,8 @@ Official NuGet packages: http://www.nuget.org
 Pre-release NuGet packages: https://vside.myget.org/gallery/devcore
 
 **Visual Studio 2019**
+* 16.1.xxx
+  * The FullPath property value for folder nodes will now always have a trailing slash (Fixes https://github.com/dotnet/project-system/issues/4485)
```

Comment:
Can we make it a little bit more solid on what has changed, maybe something like DTE folderItem.Properties["FullPath"] now always have a trailing slash? Because CPS has lots of APIs to expose FullPath property in different places, and it is hard to know what is changed.

Reply:
I actually left it vague _because_ there are lots of ways to get FullPath, and they now all return a trailing slash, but I'll add the DTE info as a sample